ladybug/designday.py

```diff
@@ -1273,11 +1357,17 @@ def __repr__(self):
 class OriginalClearSkyCondition(SkyCondition):
     """An object representing an original ASHRAE Clear Sky.
 
-    Properties:
+    Args:
         month
         day_of_month
         clearness
         daylight_savings_indicator
+
```

Comment:
I see I forgot the docstring here. Can you add the following:
```
        month: Month in which the design day occurs.
        day_of_month: Day of the month on which the design day occurs.
        clearness: Value between 0 and 1.2 that will get multiplied by the model's
            irradinace to correct for factors like elevation.
        daylight_savings_indicator: Text ('Yes' or 'No'), for whether daylight savings
            time is active. Default: 'No'
```

Reply:
Done!